protected/humhub/modules/user/models/fieldtype/Template.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace humhub\modules\user\models\fieldtype;
+
+use humhub\helpers\ArrayHelper;
+use humhub\modules\user\models\ProfileField;
+use Twig\Environment;
+use Twig\Loader\ArrayLoader;
+use Yii;
+
+/**
+ * Template is a virtual profile field that combines other profile fields using a Twig template.
+ *
+ * @package humhub\modules\user\models\fieldtype
+ * @since 1.17
+ */
+class Template extends BaseType
+{
+    public $template;
+
+    public $type = 'text';
+
+    public $isVirtual = true;
+
+    public function rules()
+    {
+        return [
+            [['template'], 'string', 'max' => 255],
+        ];
+    }
+
+    public function getUserValue($user, $raw = true): ?string
+    {
+        $variables = ArrayHelper::map(
+            $user->profile->getProfileFields(null, [static::class]),
+            'internal_name',
+            function (ProfileField $profileField) use ($user) {
+                return $profileField->getUserValue($user);
+            },
+        );
+
+        $twig = new Environment(new ArrayLoader([]));
+
```

Comment:
Can you please add a Sandbox and a SecurityPolicy. See example below. Otherwise the user (template editor) might be able to execute various PHP code.

https://github.com/humhub/humhub/blob/4ee69a1740ef423320154f1a989a5a0bef186222/protected/humhub/modules/admin/widgets/TrackingWidget.php#L39

Reply:
@luke- done